File: day-00-example/solutions/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	var sumAll int64 = 0
	var sumOdd int64 = 0
	for scanner.Scan() {
		line := scanner.Text()
		num, _ := strconv.ParseInt(line, 10, 64)
		sumAll += num

		if num%2 != 0 {
			sumOdd += num
		}
	}

	fmt.Println(sumAll)
	fmt.Println(sumOdd)
}

File: day-00-example/solutions/main.py
#!/usr/bin/env python3
#-*- coding:utf-8 -*-

import fileinput

sum_all = 0
sum_odd = 0

for line in fileinput.input():
  num = int(line)
  sum_all += num
  sum_odd += num if num % 2 != 0 else 0

# = Answer
print(str(sum_all))
print(str(sum_odd))
# ================
